apps/rush-lib/src/cli/utilities/ChangelogGenerator.ts

```diff
@@ -211,6 +215,10 @@ export default class ChangelogGenerator {
         'Patches',
         entry.comments[ChangeType[ChangeType.patch]]);
 
+      comments += ChangelogGenerator._getChangeComments(
```

Comment:
can you add if (hotFixSwitch on) around it? So that if there is any hotfix but switch is off, those change logs are not generated.

Reply:
It is optional. You do not have to do it.